src/fetch_products.py

```diff
@@ -80,60 +76,58 @@ async def validate_image_url(
     return False, {}
 
 
-async def process_category(category: str, locale: str) -> Tuple[List[Dict], List[str]]:
-    """Process products for a given category and return valid products and ASINs"""
-    valid_asins = []
+async def process_category(category: str, locale: str) -> int:
```

Comment:
@git-yogesh valid asin's are being store in the file in the process_category function correct?